### ISSUE TITLE
Crash when used with Reika's Dragon API?

### ISSUE THREAD

**Issue by WoolenSocks** [jaredlll08/Fluxed-Trinkets#5]
Hey, my game seems to crash whenever I try to add Fluxed trinkets to my pack (SSP for now while testing).

I have no idea why since the crashlog seems to have very little to no mention of your mod but the pack seems to work well without it so I figured I'd paste the log here anyways :)

http://pastebin.com/j72Ky7nW

**Comment by jaredlll08** [comment]
hey, great to see you again (we used to play Legion/Iridium-Aspect together, or at least there was someone called woolenSocks in the skype group... never spoke though...) anyway, this crash log cannot help me at all :/ what were you doing at the time? was it on startup? some more information would be really nice :)

**Comment by WoolenSocks** [comment]
Yeah I thought I recognized you!

Hmm yeah the crashlog is rather strange. It's a startup crash. If I remove the 4 reika mods I use (DragonAPI, rotary/reactor/electriCraft) it loads fine

**Comment by jaredlll08** [comment]
It has to do with rotary craft, but it isnt me causing it... So i dont know
why my mod effects it, so what are you up to these days, Iridiun-Aspect is
still going good btw

**Comment by jaredlll08** [comment]
and from what i can see you are out of a server now?(http://www.reddit.com/r/feedthebeast/comments/2bvdjr/so_botania_let_me_do_this)
 You are always welcome back btw

**Comment by WoolenSocks** [comment]
I've played a few custom packs as well as TPPI/BTP but I figured I'd assemble a pack to play a SSP play through or something alongside a server if I find something I enjoy. Or I might take a break from MC for a while and see what MS changes :P

What pack are you guys using nowadays? I really enjoyed my staying at iridium-Aspect :)
Couldn't find any updated info on the site, or maybe I'm at an old website?
